python/lsst/afw/geom/ellipses/__init__.py

```diff
@@ -23,23 +23,20 @@
 """lsst.afw.geom.ellipses
 """
 from __future__ import absolute_import
-from builtins import object
 from .ellipsesLib import *
```

Comment:
Isn't this *exactly* the case (an `__init__.py` file that brings in wrapped C++ classes) that the exception is for?